clusterloader2/pkg/prometheus/manifests/default/prometheus-serviceMonitorLegacyKubeDNS.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
What happens if we have no kube-dns in the system?

Will clusterloader block on waiting for this servicemontior to be healthy?

Reply:
No, it will just go on.

Logs from presubmit confirm that - we are using CoreDNS, and ClusterLoader is not blocked.